vere/sist.c

```diff
@@ -599,6 +599,151 @@ _sist_zest()
   u3_raft_play();
 }
 
+/* _sist_rest_nuu(): upgrade log from previous format.
+*/
+static void
+_sist_rest_nuu(u3_ulog* lug_u, u3_uled led_u, c3_c* old_c)
+{
+  c3_c    nuu_c[2048];
+  u3_noun roe = u3_nul;
+  c3_i    fid_i = lug_u->fid_i;
+  c3_i    fud_i;
+  c3_i    ret_i;
+  c3_d    end_d = lug_u->len_d;
+
+  uL(fprintf(uH, "rest: converting log from prior format\n"));
+
+  c3_assert(led_u.mag_l == u3r_mug('g'));
+
+  ret_i = snprintf(nuu_c, 2048, "%s/.urb/ham.hope", u3_Host.dir_c);
```

Comment:
I'm guessing we'd need to move ham.hope to egz.hope to get urbit to run off it?

Reply:
I think that's the `rename` step at the end of the function.  This way it doesn't corrupt your log if you kill it in the middle.